src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseLoader.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+
+[assembly: LoadableClass(DatabaseLoader.Summary, typeof(DatabaseLoader), null, typeof(SignatureLoadModel),
+    "Database Loader", DatabaseLoader.LoaderSignature)]
+
+namespace Microsoft.ML.Data
+{
+    public sealed partial class DatabaseLoader : IDataLoader<Func<DbDataReader>>
+    {
+        internal const string Summary = "Loads data from an DbDataReader.";
```

Comment:
`an DbDataReader` should be `a DbDataReader`

Reply:
Fixed.